### PR TITLE
Zoom domain name verification

### PR DESCRIPTION
**Pull request by debenedictis**
Adding verification for Zoom.

### CODE REVIEW

**Review by farski — APPROVED** [review]

**Comment by debenedictis** [comment]
@farski It's been a week but Zoom is [still verifying](https://zoom.us/account). Zoom said: The verification process takes approximately one hour to complete.

I have requested a manual approval from Zoom.

**Comment by farski** [comment]
@debenedictis This pull request was approved 8 days ago, but was never merged so it hasn't been deployed. Once a PR is approved, the person who opened the PR can merge it when they see fit. Once this gets merged I can deploy the change.

**Comment by debenedictis** [comment]
I deleted the manual approval request and updated the TXT.

**Pull request merged**
